### ISSUE TITLE
Intro Text: Gold with focus indicator does not visualize well

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#62]
The gold focus indicator is hard to see on the gold intro text box. HOWEVER, this is not a failure of the SC.
